docs/presentations/linkedin-meetup-2024-04-24.qmd

```diff
@@ -0,0 +1,549 @@
+---
+title: "Ibis"
+subtitle: "Portable Python DataFrames"
+author:
+  - Chloe He
+  - Cody Peterson
```

Comment:
remove me? would like to call in but not presenting anything

Reply:
Mostly here because the content is based on the other pres.